test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -118,40 +118,81 @@ class TestQuantizeDBRIndividualOps(QuantizeDBRTestCase):
+            m = M(dim).eval()
+            qconfig = torch.quantization.default_qconfig
+            data = self.img_data_dict[dim][0][0]
+            self._test_auto_tracing(m, qconfig, (data,))
 
     def test_conv_functional(self):
+        convs = {1: F.conv1d, 2: F.conv2d, 3: F.conv3d}
 
         class M(torch.nn.Module):
-            def __init__(self, weight2d, bias2d):
+            def __init__(self, dim, weight):
                 super().__init__()
-                self.weight2d = torch.nn.Parameter(weight2d)
-                self.bias2d = torch.nn.Parameter(bias2d)
-                self.stride2d = (1, 1)
-                self.padding2d = (0, 0)
-                self.dilation2d = (1, 1)
+                self.dim = dim
+                self.weight = torch.nn.Parameter(weight)
+                self.stride = (1,) * dim
+                self.padding = (0,) * dim
+                self.dilation = (1,) * dim
                 self.groups = 1
 
             def forward(self, x):
-                x = F.conv2d(
-                    x, self.weight2d, self.bias2d, self.stride2d, self.padding2d,
-                    self.dilation2d, self.groups)
+                x = convs[dim](
+                    x, self.weight, None, self.stride, self.padding,
+                    self.dilation, self.groups)
                 return x
 
-        model_fp32 = M(torch.randn(1, 1, 1, 1), torch.randn(1)).eval()
-        qconfig = torch.quantization.default_qconfig
-        self._test_auto_tracing(model_fp32, qconfig, (torch.randn(1, 1, 2, 2),))
+        for dim in range(1, 4):
+            data = {
+                1: torch.randn(1, 3, 10),
+                2: torch.randn(1, 3, 10, 10),
+                3: torch.randn(1, 3, 5, 5, 5)
+            }
+            model_fp32 = M(dim, data[dim]).eval()
+            qconfig = torch.quantization.default_qconfig
+            self._test_auto_tracing(model_fp32, qconfig, (data[dim],))
+
+    def test_conv_transpose_functional(self):
+        convs = {1: F.conv_transpose1d, 2: F.conv_transpose2d, 3: F.conv_transpose3d}
+
+        class M(torch.nn.Module):
+            def __init__(self, dim, weight):
+                super().__init__()
+                self.dim = dim
+                self.weight = torch.nn.Parameter(weight)
+                self.stride = 1
+                self.padding = 0
+                self.output_padding = 0
+                self.groups = 1
+                self.dilation = 1
+
+            def forward(self, x):
+                x = convs[dim](
+                    x, self.weight, None, self.stride, self.padding,
+                    self.output_padding, self.groups, self.dilation)
+                return x
+
+        for dim in range(1, 4):
+            weights = {
+                1: torch.randn(1, 3, 10),
+                2: torch.randn(1, 3, 10, 10),
+                3: torch.randn(1, 3, 5, 5, 5)
+            }
+            data = dict([(k, torch.transpose(v, 0, 1)) for k, v in weights.items()])
```

Comment:
can we just spell out weights and data, without using a formula?  A good thing to optimize for here is readability, lines of code and code reuse in this case don't seem as important.  Ideally the code is dead simple to understand for someone who is trying to fix something, copy-pasting a few lines is okay.